src/Language/PureScript/CoreImp/Optimizer/Unused.hs

```diff
@@ -14,8 +14,11 @@ removeCodeAfterReturnStatements :: AST -> AST
 removeCodeAfterReturnStatements = everywhere (removeFromBlock go)
   where
   go :: [AST] -> [AST]
-  go jss | not (any isReturn jss) = jss
-         | otherwise = let (body, ret : _) = break isReturn jss in body ++ [ret]
+  go jss =
+    case break isReturn jss of
```

Comment:
This change was necessitated by the fact that `-Wincomplete-uni-patterns` was added to `-Wall` at some point between 8.10.7 and 9.2.2; the `let (body, ret : _) = ...` is an incomplete pattern because it does not account for what happens if the second element of the tuple is an empty list. Thankfully we can avoid it (as well as an unnecessary traversal of the list) by rewriting the function to use `break` instead.